sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamComplexTypeTest.java

```diff
@@ -367,21 +370,21 @@ public void testNullInnerRow() {
         pipeline
             .apply(Create.of(row))
             .setRowSchema(nullableNestedRowWithArraySchema)
-            .apply(
-                SqlTransform.query(
-                    "select PCOLLECTION.field3.inner_row_field.string_field as string_field, PCOLLECTION.field3.array_field[1].long_field as long_field from PCOLLECTION"));
+            .apply(SqlTransform.query("select field1 is null, field3 is null from PCOLLECTION"));
 
-    PAssert.that(outputRow)
-        .containsInAnyOrder(
-            Row.withSchema(
-                    Schema.builder()
-                        .addNullableField("string_field", FieldType.STRING)
-                        .addInt64Field("long_field")
-                        .build())
-                .addValues(null, 1000L)
-                .build());
+    outputRow.apply("output", ParDo.of(new SimplePardo<Row, Integer>())).setCoder(VarIntCoder.of());
 
-    pipeline.run().waitUntilFinish(Duration.standardMinutes(2));
+    //    PAssert.that(outputRow)
```

Comment:
Was this change intentional?

Reply:
Sorry It is my bad. I fixed style issues and squash into one commit  @ibzib